CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing
+
+This doc explains how to get started contributing to [baikonur-oss](https://github.com/baikonur-oss) project.
```

Comment:
IMHO official project name should be `Baikonur`. Thus, the link should be either `Baikonur` or `Baikonur OSS` (first preferred).